pkg/registry/extensions/rest/thirdparty_controller.go

```diff
@@ -99,18 +113,18 @@ func (t *ThirdPartyController) syncResourceList(list runtime.Object) error {
 	}
 	// deletion phase, get all installed RESTful resources
 	installed := t.master.ListThirdPartyResources()
-	for _, installedAPI := range installed {
+	for _, installedGvr := range installed {
 		found := false
 		// search across the expected restful resources to see if this resource belongs to one of the expected ones
-		for _, apiPath := range existing.List() {
-			if installedAPI == apiPath || strings.HasPrefix(installedAPI, apiPath+"/") {
+		for gvr := range existing {
+			if reflect.DeepEqual(installedGvr, gvr) {
 				found = true
 				break
 			}
 		}
 		// not expected, delete the resource
 		if !found {
-			if err := t.master.RemoveThirdPartyResource(installedAPI); err != nil {
+			if err := t.master.RemoveThirdPartyResource(installedGvr); err != nil {
```

Comment:
I see the fake for this changed, and the interface, but I can't seem to find where the actual impl changed.